x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -68,16 +68,22 @@ export const getCspRules = (
 export const createRulesConfig = (
   cspRules: SavedObjectsFindResponse<CspRule>
 ): CspRulesConfiguration => {
-  const activatedRules = cspRules.saved_objects.filter((cspRule) => cspRule.attributes.enabled);
+  const benchmarkType = cspRules.saved_objects[0]?.attributes.metadata.benchmark.id;
```

Comment:
`createRulesConfig` should support multiple activated benchmarks
Hypothetically it can get multiple rules from the different benchmarks and should produce the relevant configuration

For example, we can have one rule from cis eks and another rule from cis k8s
this should produce the following configuration:

```yaml
activated_rules:
  cis_eks:
    - cis_1_1_1
  cis_k8s:
    - cis_1_1_1
```

Reply:
we currently create the rules for a single benchmark type, so an integration could only have rules from either `cis_eks` or `cis_k8s`, not both. (see https://github.com/elastic/kibana/pull/135798)